### PR TITLE
MLDB-1058: reduced string allocations/copies by using std::move on st…

### PR DESCRIPTION
**Pull request by wsourdeau**
…ring parameters

### CODE REVIEW

File: http/http_socket_handler.h [hunk 1]
```diff
@@ -1,6 +1,4 @@
-// This file is part of MLDB. Copyright 2015 Datacratic. All rights reserved.
-
-/* http_socket_handler.h                                           -*- C++ -*-
+/* http_socket_handler.h - This file is part of MLDB               -*- C++ -*-
```
Comment (guyd) [inline http/http_socket_handler.h]: I am pretty sure we want to keep the copyright notice.

Comment (wsourdeau) [inline http/http_socket_handler.h]: It's still present below. But I concede the style is not consistent with the rest.

Comment (guyd) [inline http/http_socket_handler.h]: Oh right.  I did not realized it was duplicated.

**Comment by guyd** [comment]
+1

**Pull request merged**